### ISSUE TITLE
Netflx <ganesha bansa>

### ISSUE THREAD

**Issue by ganeshabansa** [datastaxdevs/appdev-week3-graphql#391]
**Name:** ganesha bansa
**Email:** bansaganesha@gmail.com
**Linkedin Profile:** <LINK>

Attach the homework screenshots below:
![image](https://user-images.githubusercontent.com/58143498/126687473-e45ad785-6b61-4501-b694-bb80d3b0aacb.png)
![image](https://user-images.githubusercontent.com/58143498/126687530-8f8d9058-1426-46b0-95f2-bfc6ac70c1e9.png)
![image](https://user-images.githubusercontent.com/58143498/126687553-96533eb1-4db5-4a54-b513-7a4c6de29999.png)

<SCREENSHOTS>

**Comment by SonicDMG** [comment]
Hi Ganesha, nice job getting the app deployed. I don't see your custom video though on your site. Could you please add one and take a screenshot or just post your netlify link. If you did add one please point out which one. Thank you.

**Comment by ganeshabansa** [comment]
The thing is I am new for this kind of programming.
I watched this video 3 times and followed all the instructions but I didn't No how to add movies. Give me some time to learn these things .
Or please help me out how to do that

**Comment by SonicDMG** [comment]
Of course, no worries there. To add your own custom movie you can simply edit one of the mutations you used per the instructions to add the 4 Sci-Fi videos per the workshops instructions. You can change the name or something like that and apply it. If you remember this was done via the graphQL playground. Feel free to reach out to us on Discord for more help if you need to. https://dtsx.io/discord

**Comment by ganeshabansa** [comment]
thank you I am on it
